### PR TITLE
Add python project

### PR DESCRIPTION
**Pull request by deadex-ng**
I've added a python project.

### CODE REVIEW

File: data/projects.json [hunk 1]
```diff
@@ -388,5 +388,15 @@
         "level": "intermediate",
         "site": "https://github.com/lmponcio/copia-backups",
         "tags": "python, copia-backups, scripting"
+    },
+    {
+        "name": "tcp server and client in python",
```
Comment (ming-tsai) [inline data/projects.json:393]: I couldn't find this repo on your profile

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi,
I make some comments for the commit.

File: data/projects.json (hunk 1 above)
Comment (deadex-ng) [reply data/projects.json:393]: Hello @ming-tsai,
it's a github gist.

Comment (ming-tsai) [reply data/projects.json:393]: Sorry this repo for know is only allowed the repository, I will be find out the wait to save the gists

**Pull request closed without merging**